spec/integration/session_spec.rb

```diff
@@ -642,6 +642,7 @@
 
         @session.within_window(another_popup) do
           expect(@session.html).to include('Test')
+          @session.execute_script('window.close()')
```

Comment:
`another_popup.close()` outside the within_window block would be better

Reply:
fixed